operator/v1/types_dns.go

```diff
@@ -70,6 +70,24 @@ type DNSSpec struct {
 	// DNS
 	// +optional
 	ManagementState ManagementState `json:"managementState,omitempty"`
+
+	// operatorLogLevel controls the logging level of the DNS Operator.
+	// Valid values are: "Normal", "Debug", "Trace".
```

Comment:
https://github.com/openshift/enhancements/pull/931 says that "TraceAll" is also valid.